stages/org.osbuild.users

```diff
@@ -125,6 +126,10 @@ def main(tree, options):
         else:
             useradd(tree, name, uid, gid, groups, description, home, shell, password, expiredate)
 
+
```

Comment:
The linter doesn't like the extra empty line here :)

Reply:
Done!